### PR TITLE
Update Docker build procedure + Documentaiton

### PR DESCRIPTION
**Pull request by dmitri-mcguckin**
* Rework Docker image to deploy without the use of Maven
* Add `/prod` folder to `/etc` to enable prod-specific config overrides (allows the disabling of security module in dev-environment + more)
* Extensive reorganizing of the MDB into sub-categories
* Reorganization of Java sub-packages inside `org.oresat.uniclogs`

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    branches:
+      - $default-branch
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build image
+        run: docker-compose -f docker-compose.prod.yaml build
```
Comment (ryanpdx) [inline .github/workflows/docker.yaml]: This docker-compose file was deleted.

File: README.md [hunk 2]
```diff
@@ -5,28 +5,129 @@
...
-- See [docker/README.md](docker/README.md)
+#### Pull the latest image from the official OreSat image via Dockerhub:
 
+`docker pull oresat/uniclogs-yamcs:latest`
 
-## Command Shell
+#### Run immediately in detatched mode:
+
+```
+docker run \
```
Comment (ryanpdx) [inline README.md:27]: Why not just just docker-compose?

**Review by ryanpdx — COMMENTED** [review]
Looks good

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (dmitri-mcguckin) [reply .github/workflows/docker.yaml]: Disabled workflow, can recreate it with DiiD later for CI image building.

**Review by ryanpdx — COMMENTED** [review]
Why re-add the AUR configs? Isn't the plan to move to docker only?

**Comment by dmitri-mcguckin** [comment]
Just so we don't lose it.

The intention is to move to Docker deployment eventually but we're not there yet.

Still need the CI automation and other small fixes.

Not to mention, the AUR stuff will be a nice reference if we make more AUR packages down the road.

**Review by ryanpdx — APPROVED** [review]
LGTM

**Pull request merged**